packages/ckeditor5-email/src/integrations/font.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/**
+ * @module email/integrations/font
+ */
+
+import { Plugin } from 'ckeditor5/src/core.js';
+import type { FontColorConfig } from '@ckeditor/ckeditor5-font';
+
+import EmailIntegrationUtils from '../emailintegrationutils.js';
+
+/**
+ * A plugin that checks if the Font plugin is properly configured for the email integration.
+ */
+export class FontIntegration extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	public static get requires() {
+		return [ EmailIntegrationUtils ] as const;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static get pluginName() {
+		return 'FontIntegration' as const;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static override get isOfficialPlugin(): true {
+		return true;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public afterInit(): void {
+		const { plugins } = this.editor;
+
+		if ( plugins.has( 'Font' ) ) {
```

Comment:
I'd check a specific plugin as 'Font' is just a glue plugin and features might be added separately.